src/components/Sidebar/Sidebar.component.jsx

```diff
@@ -0,0 +1,16 @@
+import './Sidebar.css';
+
+const Sidebar = ({data, title, filters, clickEvent}) => {
+    return (
+        <div>
+            <h2>{title}</h2>
+            <ul style={{listStyle: "none"}}>
+            {data.map( obj => {
+               return (<li key={obj.id} className={`${filters.includes(obj.data.name.toLowerCase()) ? "enabled" : "disabled"}`} onClick={() => clickEvent(obj.data.name.toLowerCase())}>{obj.data.name}</li>) 
```

Comment:
This and in the filter I think its where you should make the change to make it more intuitive